api/src/public/login/script.js

```diff
@@ -152,8 +154,13 @@ const parseTranslations = function() {
   return JSON.parse(decodeURIComponent(raw));
 };
 
+const checkUsername = function() {
+  const usernameQueryParam = urlParams.get('username');
+  const usernameEntered    = document.getElementById('user').value.toLowerCase().trim();
```

Comment:
Instead of duplicating the username getting code, let's pull out a function for this, or pass it in as a parameter.

Reply:
I passed user as a parameter of the function. Thanks.